examples/imgui_impl_sdl.cpp

```diff
@@ -236,13 +244,21 @@ static void ImGui_ImplSDL2_UpdateMousePosAndButtons()
     SDL_Window* focused_window = SDL_GetKeyboardFocus();
     if (g_Window == focused_window)
     {
-        // SDL_GetMouseState() gives mouse position seemingly based on the last window entered/focused(?)
-        // The creation of a new windows at runtime and SDL_CaptureMouse both seems to severely mess up with that, so we retrieve that position globally.
-        int wx, wy;
-        SDL_GetWindowPosition(focused_window, &wx, &wy);
-        SDL_GetGlobalMouseState(&mx, &my);
-        mx -= wx;
-        my -= wy;
+        if (g_SDL_VIDEODRIVER_IS_WAYLAND)
+        {
+            // on Wayland the GetGlobalMouseState always returns 0,0
+            SDL_GetMouseState(&mx, &my);
+        }
```

Comment:
This block already exists in the function, see whole function, should only have an `#if defined()` at the top of the block likely.

Reply:
removed it